pkg/querier/worker/scheduler_processor.go

```diff
@@ -213,7 +215,9 @@ func (sp *schedulerProcessor) runRequest(ctx context.Context, logger log.Logger,
 			Body: []byte(errMsg),
 		}
 	}
+	retries := 0
 
+retry:
```

Comment:
Doesn't seem like the label + `goto` are really needed and the same thing could be accomplished with a `for` loop.

Reply:
changed @ 51512047b4d488e3d7aa25e20c2fad8fd10d82ab